src/types/version_number.rs

```diff
@@ -66,4 +66,11 @@ impl VersionNumber {
         }
         false
     }
+
+    pub fn is_major_update(&self, other: &VersionNumber) -> bool {
```

Comment:
Please add an comment above the function. Otherwise the pipeline will fail...

Reply:
Hi @cophilot  ,Added Comment in the implementation function in this [commit](https://github.com/cophilot/templify/pull/12/commits/1dfe74d7a60718119744a66e6eee16684293f344). 
Can you please help me in finding out the protocols that I should follow further, so that pipile will never fail.
Thankyou

Reply:
@RounakJoshi09 
I still have to write some documentation about that, I just have been to lazy :D But in general the Pipeline will check the code in regards of functionality and quality and will lint, format and run test on the code base. You can take a look at the [check_rust_quality file](https://github.com/cophilot/templify/blob/master/.github/workflows/check_rust_quality.yml) to see what will be running in the pipeline.